apps/calendar/js/views/week.js

```diff
@@ -31,7 +31,7 @@ WeekView.prototype = {
     }
 
     // Show monday as the first day of the grid if date is between Mon-Fri.
-    var index = Calc.dayOfWeekFromMonday(date.getDay());
+    var index = Calc.dayOfWeekFromStartDay(date.getDay());
     if (index < 5) {
```

Comment:
it doesn't work. need to keep the old `dayOfWeekFromMonday` logic or implement something similar here (we want to set the base date as monday if date is between mon-fri)